src/test/java/teammates/test/cases/storage/InstructorsDbTest.java

```diff
@@ -314,12 +314,24 @@ public void testUpdateInstructorByGoogleId() throws Exception {
 
         instructorToEdit.name = "New Name";
         instructorToEdit.email = "InstrDbT.new-email@email.tmt";
+        instructorToEdit.isArchived = true;
+        instructorToEdit.role = "new role";
+        instructorToEdit.isDisplayedToStudents = false;
+        instructorToEdit.displayedName = "New Displayed Name";
+        instructorToEdit.privileges = new InstructorPrivileges(
+                Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_OBSERVER);
         instructorsDb.updateInstructorByGoogleId(instructorToEdit);
 
         InstructorAttributes instructorUpdated =
                 instructorsDb.getInstructorForGoogleId(instructorToEdit.courseId, instructorToEdit.googleId);
         assertEquals(instructorToEdit.name, instructorUpdated.name);
         assertEquals(instructorToEdit.email, instructorUpdated.email);
+        assertTrue(instructorUpdated.isArchived);
+        assertEquals("new role", instructorUpdated.role);
+        assertFalse(instructorUpdated.isDisplayedToStudents);
+        assertEquals("New Displayed Name", instructorUpdated.displayedName);
+        assertTrue(instructorUpdated.hasObserverPrivileges());
+        assertFalse(instructorUpdated.hasManagerPrivileges());
```

Comment:
Would this be okay:
```JAVA
assertTrue(instructorUpdated.hasObserverPrivileges());
// Verifying less privileged 'Observer' role did not return false positive in case old 'Manager' role is unchanged.
assertFalse(instructorUpdated.hasManagerPrivileges());
```